src/App.js

```diff
@@ -1,11 +1,32 @@
 import React, { Component } from 'react';
-import './css/App.scss';
 import Card from './components/Card';
+import './css/components/App.css';
 
 class App extends Component {
+  constructor(props) {
+    super();
+
+    this.state = {
+      theme : '' // example: use 'monkey' or 'minimal'
+    }
+  }
+
+  componentDidMount() {
+    // Checks for theme and loads if needed
+    if (this.state.theme) {
+      try {
+        require(`./css/themes/${this.state.theme}.css`);
```

Comment:
Using require in the middle of the code is not a good practice. Let's figure out how to do this better

Reply:
What's the issue exactly? Is it the issue using 'require' as an instruction or the fact we're loading resources in the middle of the code?

Reply:
Also, would it be better to require that conditionally in the constructor? Any resources?

Reply:
I'm going ahead, merging this and open a new branch to refactor and include react-helmet to handle this and a few more things :)